website/docs/install/install-with-bazel.md

```diff
@@ -66,11 +78,11 @@ cd prysm
 3. Build both the beacon chain node and the validator client:
 
 ```text
-bazel build //beacon-chain:beacon-chain --config=release
-bazel build //validator:validator --config=release
+bazelisk build //beacon-chain:beacon-chain --config=release
```

Comment:
After doing the alias approach mentioned above, there is no need to use `bazelist` because `bazel` will work just the same